syntax = "proto3";

package PublicProtocol;

message BaseRoleInfo {
    uint64 role_id = 1;
    bytes name = 2;
    bytes avatarurl = 3;
    bool online = 4;
    uint64 logoutts = 5;
}